internal/pkg/mount/v2/mount.go

```diff
@@ -290,7 +299,15 @@ func (p *Point) Move(newTarget string) error {
 }
 
 func (p *Point) mount() error {
-	return unix.Mount(p.source, p.target, p.fstype, p.flags, p.data)
+	if err := unix.Mount(p.source, p.target, p.fstype, p.flags, p.data); err != nil {
+		return err
+	}
+
+	if p.selinuxLabel != "" {
+		return selinux.SetLabel(p.target, p.selinuxLabel)
+	}
```

Comment:
SetLabel checks that on its own

```suggestion
	return selinux.SetLabel(p.target, p.selinuxLabel)
```